CMakeLists.txt

```diff
@@ -497,14 +497,36 @@ if(BUILD_DOCS)
     install(FILES "README.html" DESTINATION "${DFHACK_DATA_DESTINATION}")
 endif()
 
-option(BUILD_TESTS "Include tests (currently just installs Lua tests into the scripts folder)" OFF)
-if(BUILD_TESTS)
-    if(EXISTS "${dfhack_SOURCE_DIR}/test/scripts")
-        message(SEND_ERROR "test/scripts must not exist in the dfhack repo since it would conflict with the tests installed from the scripts repo.")
+# Testing with CTest
+include(CTest)
+
+include(CMakeDependentOption)
+cmake_dependent_option(
+    BUILD_TEST_SCRIPTS "Install integration tests in hack/scripts/test" OFF
```

Comment:
perhaps `BUILD_TESTING_LUA` instead of `BUILD_TEST_SCRIPTS`. `BUILD_TEST_SCRIPTS` implies (at least to me) that only the scripts are being tested. The Lua-based testing is more general than that.